MobileCenter/MobileCenter/Internals/Channel/MSLogManager.h

```diff
@@ -5,7 +5,8 @@
 #import "../Model/MSLog.h"
 #import "MSEnable.h"
 #import "MSLogManagerDelegate.h"
-#import <Foundation/Foundation.h>
+
```

Comment:
Sometimes there is a space between imports sometimes not, we may want to harmonize this as much as possible.